pkgs/applications/video/epgstation/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, pkgs, makeWrapper, bash, nodejs }:
+
+let
+  nodePackages = import ./node-composition.nix {
+    inherit pkgs nodejs;
+    inherit (stdenv.hostPlatform) system;
+  };
+  workaround-opencollective-buildfailures = stdenv.mkDerivation {
+    # FIXME: This should be removed when a complete fix is available
+    # https://github.com/svanderburg/node2nix/issues/145
+    name = "workaround-opencollective-buildfailures";
+    phases = [ "installPhase" ];
+    installPhase = ''
+      mkdir -p $out/bin
+      touch $out/bin/opencollective-postinstall
+      chmod +x $out/bin/opencollective-postinstall
+    '';
+  };
+in
+nodePackages."epgstation-git+https://github.com/l3tnun/EPGStation.git#v1.7.4".override (drv: {
```

Comment:
Is the version hardcoded here on purpose?

Reply:
Yes, it seems to be the way node2nix works for git urls. I couldn't refer to the Epgstation derivation without specifying the full version.

Reply:
This is no longer  the case as the details have changed.